.gitignore

```diff
@@ -8,6 +8,7 @@ cscope.po.out
 cscope.out
 smf/manifests/*.xml
 .DS_Store
+.idea
```

Comment:
done that because of 

"Which file to place a pattern in depends on how the pattern is meant to be used.

Patterns which should be version-controlled and distributed to other repositories via clone (i.e., files that all developers will want to ignore) should go into a .gitignore file."

from http://git-scm.com/docs/gitignore